make/photon/registry/entrypoint.sh

```diff
@@ -2,12 +2,11 @@
 
 set -e
 
-if [ -d /etc/registry ]; then
-    chown 10000:10000 -R /etc/registry
-fi
 if [ -d /var/lib/registry ]; then
+    # For CI testng.
```

Comment:
please add detail description for this chown, and 'testng' is a typo.

Reply:
yep, will do.